lib/ret/hub.ex

```diff
@@ -123,6 +128,20 @@ defmodule Ret.Hub do
     end
   end
 
+  def ensure_room!(hub) do
+    if RoomAssigner.is_alive?(hub.host) do
+      hub
+    else
+      host = RoomAssigner.get_available_host(hub.host)
+
+      if host && host != hub.host do
+        hub |> changeset_for_new_host(host) |> Repo.update!()
+      else
+        hub
```

Comment:
Maybe we should actually throw in this case, since this is `ensure_room!`?

Reply:
ah actually I think we should fall through as a worst-case-scenario type of situation (perhaps the supervisor census is dead, but janus is still alive) but we can probably drop the `!` from the name